contributors/design-proposals/cpu-manager.md

```diff
@@ -266,14 +270,22 @@ func (p *staticPolicy) UnregisterContainer(s State, containerID string) error {
        cpuset for all containers running in the shared pool.
 
 1. _The shared pool becomes empty._
-    1. The CPU manager adds a node condition with effect NoSchedule,
-       NoExecute that prevents BestEffort and Burstable QoS class pods from
-       running on the node. BestEffort and Burstable QoS class pods are
-       evicted from the node.
+    1. The CPU manager sets a CPUPressure node condition to true that prevents
+       BestEffort and Burstable QoS class pods with no CPU resource request
+       from being scheduled on the node. Already running BestEffort and Burstable QoS
+       class pods without a CPU resource request are evicted from the node.
+
+       NOTE: The decision to allow the shared pool to completely empty is
+       driven by a desire to keep the scheduler accounting simple. If a
+       --cpu-policy-static-min-shared-cpus flag were to exist, there is no simple way
+       to convey that information to the scheduler. The scheduler would then need to
+       know that some portion of the CPU Allocatable is "special", for use only by
+       non-exlusive containers.
```

Comment:
Possibly obvious, but consider also mentioning what would happen with a shared pool low-water-mark but without scheduler enhancements -- Kubelet would need to reject some G pods after scheduling, with high potential to confuse users.